blocks/card/card.css

```diff
@@ -0,0 +1,128 @@
+.card-box {
+  box-sizing: border-box;
+  border: 1px solid #dadada;
+  background-color: var(--color-background);
+  display: flex;
+  flex-direction: column;
+  border-radius: 1rem;
+  box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.1),
```

Comment:
Let's use `rem` throughout the CSS.

We also have a variable for `box-shadow`! The variables live in `base.css` - use the `--drop-shadow-emphasized` value here.